js/app.js

```diff
@@ -18,17 +16,13 @@ var Ingredients = {
     'fruity': ['slice of orange', 'dash of cassis', 'cherry on top']
 };
 
-//not sure why you would need this object in this example...
```

Comment:
ingredients should be within the pantry, possibly
`var pantry = function(ingredients) {
    this.ingredients = ingredients
}`